src/og/control/DebugInfo.ts

```diff
@@ -52,17 +50,16 @@ const ICON_BUTTON_SVG = `<?xml version="1.0" encoding="iso-8859-1"?>
 </g>
 </svg>`;
 
-/**
- * Debug information
- * @class
- * @extends {Control}
- * @param {Object} [options] - Control options.
- */
-class DebugInfo extends Control {
+export class DebugInfo extends Control {
+    el: any;
+    _watch: any;
+    _toggleBtn: ToggleButton;
+    _dialog: Dialog;
+
     /**
      * @param {Object} [options] - Control options.
      */
-    constructor(options = {}) {
+    constructor(options: { name?: string, watch?: any } = {}) {
```

Comment:
Let's add an interface, `IDebugInfoParams extends IControlParam { ... }`

Reply:
Can I do this in a follow up PR?